source/routes/fhir-r4.rst

```diff
@@ -0,0 +1,57 @@
+FHIR R4
+*******
+
+Purpose
+========
+The Linux for Health (LFH) FHIR R4 route allows you to send FHIR R4 data to Linux for Health, store that data as part of the LFH  Longitudinal Patient Record (LPR) and notify listeners of the stored data via NATS, for downstream integration.
+
+Details
+=======
++-------------------------+---------------------------------------------------------------------+
+| Parameter               | Description                                                         |
++=========================+=====================================================================+
+| URL Property            | linuxforhealth.connect.endpoint.fhir_r4_rest.baseUri                |
++-------------------------+---------------------------------------------------------------------+
+| Default URL             | http://0.0.0.0:8080/fhir/r4/{resource}                              |
++-------------------------+---------------------------------------------------------------------+
+| Example URL             | http://0.0.0.0:8080/fhir/r4/Patient                                 |
++-------------------------+---------------------------------------------------------------------+
+| Protocol                | REST                                                                |
++-------------------------+---------------------------------------------------------------------+
+| Supported Operation(s)  | POST                                                                |
++-------------------------+---------------------------------------------------------------------+
+
+Configuration
+=============
+
+Path Parameters
+---------------
++--------------------+-----------+--------------------------------------------------------------+
+| Parameter          | Type      | Description                                                  |
++====================+===========+==============================================================+
+| resource           | Resource  | The FHIR R4 JSON resource to save.                           |
```

Comment:
Regarding the `json` reference here, it's actually the **resource type name** that's included in the path param. The resource type path param should obviously match the resource type of the json body in the request.
Picking up on the `json` reference, maybe it's worth noting that LFH only supports `application/fhir+json` in the POST at this time - i.e. you can't post an xml representation of a FHIR resource.
Are FHIR bundle resources supported?

Reply:
good point, I'll include type in the verbiage.  and I'll update to include that header.  Bundle resources are supported.